c2corg_api/views/user.py

```diff
@@ -67,15 +67,18 @@ def validate_json_password(request):
         request.errors.add('body', 'password', 'Invalid')
 
 
+def available_user_attribute(attrname, value):
+    attr = getattr(User, attrname)
+    return DBSession.query(User).filter(attr == value).count() == 0
```

Comment:
Thanks for testing! So, you were right, it doesn't make a difference. Maybe Postgres takes into account that the column is unique?